walker/tar.go

```diff
@@ -16,7 +16,7 @@ const (
 	wh  string = ".wh."
 )
 
-func WalkLayerTar(layer io.Reader, analyzeFn WalkFunc) ([]string, []string, error) {
+func WalkLayerTar(layer io.Reader, analyzeFn WalkFunc, skipDirectories ...string) ([]string, []string, error) {
```

Comment:
nit: just my preference, but I prefer a function to be at the end for readability.
```suggestion
func WalkLayerTar(layer io.Reader, skipDirectories []string, analyzeFn WalkFunc) ([]string, []string, error) {
```